src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -38,18 +40,18 @@ const FormDefinitionDetailPage = () => {
       MutationUpdateFormDefinitionArgs
     >({
       formRole: StaticFormRole.FormDefinition,
-      initialValues: formDefinition || {},
-      localConstants: { definitionId: formId },
+      initialValues: formIdentifier?.draft || {},
+      localConstants: { definitionId: formIdentifier?.draft.id },
```

Comment:
This code is causing typescript build failures since it relies on `formIdentifier` having already resolved, and can't be moved after the formIdentifier's resolution because it is a hook that can't be called conditionally.

In practice, I think this doesn't actually cause problems because the button you click to open this dialog is not visible until the formIdentifier resolves. I think this code will also change/maybe get deleted entirely when we implement https://github.com/open-path/Green-River/issues/6005, so maybe it should be ignored for now?

Reply:
yep I think your solution is fine- this comes up in various places in the code we have the same `|| ''` to pacify ts